src/device.rs

```diff
@@ -62,6 +62,34 @@ impl Device {
         Ok(Interface { backend })
     }
 
+    /// Detach kernel drivers for the specified interface.
+    ///
+    /// ### Platform notes
+    /// This function can only detach kernel drivers on Linux. Calling on other platforms has
+    /// no effect.
+    pub fn detach_kernel_driver(&self, interface: u8) -> Result<(), Error> {
+        #[cfg_attr(not(target_os = "linux"), allow(unused_variables))]
+        let interface = interface;
```

Comment:
You could `let _ = interface;` to suppress the unused argument without creating a new unused variable.